lib/landlord/landlord.controllers.js

```diff
@@ -4,9 +4,7 @@ module.exports = (models) => {
   const Sequelize = require('sequelize');
   const Op = Sequelize.Op;
   const landlordModels = require('./landlord.models');
-  const accountLib = require('../account/account.controllers')(models).lib;
```

Comment:
removing this is probably what is breaking tests. please revert changes

Reply:
This is so weird.  I don't think I made any changes to this, but maybe I accidentally removed it when thought I was looking at my controller.  I'll fix it.